src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -83,10 +88,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     /**
      * Loads the Event Dispatcher from the container.
      *
+     * @param string $serviceId
+     *
      * @return EventDispatcherInterface
      */
-    protected function getEventDispatcher()
+    protected function getEventDispatcher($serviceId)
     {
-        return $this->getContainer()->get('event_dispatcher');
+        $eventDispatcher = $this->getContainer()->get($serviceId);
+
+        if (false === $eventDispatcher instanceOf EventDispatcherInterface) {
```

Comment:
In the Symfony code base, a more common form of checking if an object doesn't implement an interface is:

```
if (!$eventDispatcher instanceOf EventDispatcherInterface) {
}
```